src/index.js

```diff
@@ -38,23 +38,23 @@ function createMCServer (options) {
 
 class MCServer extends EventEmitter {
   constructor () {
+    plugins.initPlugins()
     super()
     this._server = null
   }
 
   connect (options) {
+    this.options = options
```

Comment:
Not sure why it wasn't exposed before, but I need to change options later (that are used in plugins).

Reply:
How and by what are options changed? Are you sure that happens before options are used ?

Reply:
Some plugin's options are used inside callbacks (`view-distance` is one of these)
Anyway we can already modify the original options object, this is just done for convenience, so I can also use options value like this:

```js
const server = createMCServer(...)
server.options.worldFolder // do smth with this
```

Reply:
Are you sure that happens before options are used ?

Reply:
Which options do you mean? Some options such as `view-distance` are used when the player connects for example (sendMap, sendRestMap). However, I mostly used it for reading options only. I don't mind removing it if you don't like it tho.